destinations/destinations.go

```diff
@@ -0,0 +1,34 @@
+package destinations
+
+import (
+	"io"
+	"net"
+	"os"
+)
+
+var (
+	connection io.WriteCloser
+)
+
+func NewRemote() (io.WriteCloser, error) {
+	// read remote logging destination from environment
+	destination := os.Getenv("FISKALY_REMOTE_LOGGING_DESTINATION")
+	if destination == "" {
+		panic("env var FISKALY_REMOTE_LOGGING_DESTINATION not defined.")
+	}
+
+	// open UDP socket
```

Comment:
Would probably also move this into a small function called `getConnection`

Reply:
done